test/src/e2e_vm_tests/test_programs/should_pass/language/array_basics/src/main.sw

```diff
@@ -1,32 +1,42 @@
 script;
 
 struct S {
-  foo: u64,
-  bar: u64,
+    foo: u64,
+    bar: u64,
 }
 
 fn main() -> bool {
-    let a: [bool; 5] = [true, true, true, false, true];
-    let b: [u32; 10] = [3; 10];
+    let a: [bool;
+    5] = [true, true, true, false, true];
+    let b: [u32;
```

Comment:
What's going on with the arrays here?

Reply:
This is the result of the old formatter, meaning the code wasn't formatted according to it previously. This will change when we switch over, and it will look much better (: